### DISCUSSION TITLE
Possible issue with zooz 800 stick and ring keypad v2

### DISCUSSION THREAD

**Discussion by btehan** [zwave-js/zwave-js-ui#3156] [Q&A] [answered]
I'm migrating from a HUSBZB-1 to a ZST39 and manually adding all my devices has gone smoothly except for my two ring keypad v2s.  I normally run zwave-js-ui in a LXC on proxmox with the USB stick passed through to the container.  When I go to add the keypad with S2 access control (either via smart start or normal pairing) with S2 the CPU usage on the LXC will plateau and the  zwave-js-ui webpage fails to load and all my z-wave devices disappear from HA.  If I then load up Z-Wave PC Controller in Simplicity Studio, I can see the keypad listed and remove it.  Then I move the zooz stick back to the zwave-js-ui LXC and the webpage loads just fine.

I then moved the zooz stick over to the HAOS VM and turned up the  Z-Wave JS add-on.  I copied all 4 auth keys over to the config and selected the USB stick.  The add-on booted up fine and I could see logs about it talking to devices.  I then went to the HA integrations page and hit ADD INTEGRATION >> Add Z-Wave device and was able to pair the ring keypad v2:

 Firmware: 1.18.0
 ID: 58
 Status: Alive
 Ready: Yes
 Highest security: S2 Access Control
  Z-Wave Plus: Version 2

I'm unsure of how to get logs from zwave-js-ui when I can't load the GUI.  Does this sound like a bug with zwave-js-ui?  Anyone run into something like this?

**Comment by robertsLando** [comment]
I would need some logs to understand what's going on but I'm not an HA expert, maybe @kpine @raman325 could help here telling how to get the logs from terminal?

**Comment by btehan** [comment]
I can SSH into the zwave-js-ui LXC.  Would there be a file somewhere in there with logs?  Where else does zwave-js-ui store logs outside of the GUI?

**Reply by robertsLando** [reply]
Project folder > store > logs > *.log

**Reply by btehan** [reply]
[zwave-js-ui-store.zip](https://github.com/zwave-js/zwave-js-ui/files/12090886/zwave-js-ui-store.zip)

Hey, sorry for not getting back to you.
There wasn't anything within the LXC console/ssh, but I did find some logs in the GUI.

**Reply by robertsLando** [reply, marked as answer]
@btehan Thre was an issue with keypad, wait next release or try the latest
